### PR TITLE
Improve the READMEs for the client.

### PR DESCRIPTION
**Pull request by brendandburns**
cc @ityuhui

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,72 @@
...
+cd ${CLIENT_REPO_ROOT}/examples/list_pod
+make
+```
+
+## Running the example
+For now, you need to use `kubectl proxy` to handle authentication.
+
+```bash
+kubectl proxy
+./list_por_bin
```
Comment (ityuhui) [inline README.md]: a small typo:
```
./list_pod_bin
```

**Comment by ityuhui** [comment]
Hi @brendandburns 

It's a very good document that will help user a lot !
And gave a great kick-off for the project kubernetes-client/c !

File: README.md (hunk 1 above)
Comment (brendandburns) [reply README.md]: fixed.

**Comment by brendandburns** [comment]
Thanks for catching the typo, comment addressed.

**Pull request merged**